resources/dicts/thoughts/alive/apprentice.json

```diff
@@ -19,18 +19,6 @@
             "Practices {PRONOUN/m_c/poss} highest jump",
             "Practices {PRONOUN/m_c/poss} farthest leap",
             "Feels proud of {PRONOUN/m_c/self} for doing so well in training today",
-            "Is gossiping",
-            "Is acting angsty",
-            "Is dreading {PRONOUN/m_c/poss} apprentice duties",
-            "Fell into the nearby creek yesterday and is still feeling damp",
-            "Is having a hard time keeping up with {PRONOUN/m_c/poss} training",
-            "Was tasked with lining nests with fresh moss today",
-            "Is dreaming of someday making {PRONOUN/m_c/poss} Clan proud",
-            "Is thinking about the time {PRONOUN/m_c/subject} caught a huge rabbit",
-            "Practices some battle moves",
-            "Is helping to reinforce the camp wall with brambles",
-            "Is daydreaming about having a mate and kits someday",
-            "Had quite the adventure today",
```

Comment:
Was the removal of these intentional?

Reply:
Yes - they're duplicates of content found in general with a wider scope. Confusingly because of the way these are laid out, if you want to make a general apprentice-wide thought (i.e. for warrior, medcat or mediator apprentices), it has to live in general.

Reply:
I see!  I figured it was something like that, just wanted to check

Reply:
I think that a case could be made that "practicing battle moves" should be warrior apprentice only, but I don't think it's unreasonable for all apprentices to be learning battle moves anyway just in case.